src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2864,6 +2864,7 @@ def _keyvault_update(
             raise InvalidArgumentValueError("Please provide a valid keyvault ID")
 
         if mc.ingress_profile and mc.ingress_profile.web_app_routing and mc.ingress_profile.web_app_routing.enabled:
+            cmd.command_kwargs['operation_group'] = 'vaults'
```

Comment:
Cannot tell what the change is trying to fix. Without mutating the value of `operation_group`, would keyvault_client.get fail?

Besides, it seems that the existing test case test_aks_approuting_update does not cover the scenario to update key vault, would you help update the test case?